engine/access/access_test.go

```diff
@@ -424,12 +424,53 @@ func (suite *Suite) TestGetExecutionResultByBlockID() {
 		blockID := unittest.IdentifierFixture()
 
 		er := unittest.ExecutionResultFixture(
-			unittest.WithExecutionResultBlockID(blockID),
-			unittest.WithServiceEvents(2))
```

Comment:
why did we remove this line?

Reply:
`ServiceEvent` fixture generation is non-deterministic, making parsing hard, but I can do what you suggested below and be more comprehenesive